assets/js/components/Work/Tabs/About.jsx

```diff
@@ -1,134 +1,211 @@
-          </a>
-        </h2>
-        {showCoreMetadata && (
-          <div className="box">
-            <div className="field">
-              <label className="label">Description</label>
-              <div className="control">
-                {isEditing ? (
-                  <textarea
-                    name="description"
-                    className="textarea"
-                    placeholder="e.g. Hello world"
-                    value={descriptiveMetadata.description}
-                    onChange={e => setDescription(e.target.value)}
-                  />
-                ) : (
-                  <p className="has-text-grey">
-                    {descriptiveMetadata.description ||
-                      "No description provided"}
-                  </p>
-                )}
+    <form name="work-about-form" onSubmit={handleSubmit(onSubmit)}>
+      <div className="columns is-centered" data-testid="tab-about-content">
+        <div className="column is-two-thirds">
+          <h2 className="title is-size-5">
+            Core Metadata{" "}
+            <a onClick={() => setShowCoreMetadata(!showCoreMetadata)}>
+              <FontAwesomeIcon
+                icon={
+                  showCoreMetadata
+                    ? "chevron-circle-down"
+                    : "chevron-circle-right"
+                }
+              />
+            </a>
+          </h2>
+          {showCoreMetadata && (
+            <div className="box">
+              <div className="field">
+                <label className="label">Description</label>
+                <div className="control">
+                  {isEditing ? (
+                    <textarea
+                      ref={register}
+                      name="description"
+                      className="textarea"
+                      data-testid="description"
+                      placeholder="e.g. Hello world"
+                      defaultValue={descriptiveMetadata.description}
+                    />
+                  ) : (
+                    <p>
+                      {descriptiveMetadata.description ||
+                        "No description provided"}
+                    </p>
+                  )}
+                </div>
               </div>
-            </div>
-            <div className="field">
-              <label className="label">
-                Date Created (Should this be editable?)
-              </label>
-              <div className="control">
-                {isEditing ? (
-                  <input
-                    name="insertedAt"
-                    className="input"
-                    type="date"
-                    value={insertedAt}
-                    onChange={e => setInsertedAt(e.target.value)}
-                  />
-                ) : (
-                  <p>{formatDate(work.insertedAt)}</p>
-                )}
+              <div className="field">
+                <label className="label">
+                  Date Created{" "}
```

Comment:
Date Created is not the same thing as `inserted_at`
`inserted_at` is an autogenerated database field (generated when the work is created) and is not editable. (There is an `updated_at` field that is auto generated when the record is updated)

Date Created is a field of descriptive metadata that is not yet exposed in the API. It is an EDTF date, so a regular calendar would not be appropriate as a chooser. (It will also eventually be a multi-valued field)

Reply:
Ah, good to know.  Thanks for the info @kdid